src/service/service-directory-api/src/FamilyHubs.ServiceDirectory.Data/Entities/Staging/ServicesTemp.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace FamilyHubs.ServiceDirectory.Data.Entities.Staging;
+
+public class ServicesTemp
+{
+    public long Id { get; set; }
+    public required string Json { get; set; }

```

Comment:
.NET 8 supports JSON columns directly, dunno if there's anything special we need to do with the type or if `string` is fine?

Reply:
My understanding is to do that I'd actually have to provide the full JSON type structure which I'm pretty sure is out of scope of this ticket.